bin/show_tag.rb

```diff
@@ -8,18 +8,26 @@
 require 'optimist'
 
 opts = Optimist.options do
-  opt :tag,    "The tag name",      :type => :string, :required => true
-  opt :branch, "The target branch", :type => :string, :required => true
+  opt :tag, "The tag name.", :type => :string, :required => true
+
+  ManageIQ::Release.common_options(self, :except => :dry_run, :repo_set_default => nil)
 end
+opts[:repo_set] = opts[:tag].split("-").first unless opts[:repo] || opts[:repo_set]
```

Comment:
This is an example where we want the tag name to drive the default repo set name for convenience.  In this case if a user tags `--tag jansa-1-rc1`, then we infer the repo_set is named "jansa" unless the user has specified a different repo_set _or_ a list of repos.